cg/meta/demultiplex/validation.py

```diff
@@ -6,24 +6,25 @@
 from cg.exc import FlowCellError
 
 
-def validate_sample_fastq_file(sample_fastq: Path) -> None:
+def is_valid_sample_fastq_file(sample_fastq: Path, sample_id: str) -> bool:
```

Comment:
```suggestion
def is_valid_sample_fastq_file(sample_fastq: Path, sample_internal_id: str) -> bool:
```

Reply:
Would also be nice to have an indication which assumption is violated I suppose

Reply:
Hm, yeah I removed that - there post processing flow logs a warning when a fastq file cannot be processed so I thin it is fine without for now.